general/payment_acquirers/stripe.rst

```diff
@@ -0,0 +1,32 @@
+======
+Stripe
+======
+
+`Stripe <https://stripe.com/>`_ is a United States-based online payment solution
+provider, allowing businesses to accept **credit cards** and other payment methods.
+
+Enable Local Payment Methods
+============================
+
+Local payment methods are payment methods that are only available for certain merchants
+and customers countries and currencies.
+
+The Stripe connector in Odoo supports the integration of the following local payment methods:
+
+- Bancontact
+- EPS
+- Giropay
+- iDeal:
+- Przelewy24 (P24)
```

Comment:
Is this list exhaustive?
Your screenshot suggests that there are more payment methods. If this relies on Stripe, I would advise you to link the text "local payment methods" with the page listing all methods available and rephrase your sentence.

```suggestion
The Stripe connector in Odoo supports many local payment methods, including:

- Bancontact
- EPS
- Giropay
- iDeal:
- Przelewy24 (P24)
```

Reply:
The list is indeed exhaustive. Other payment methods are not global.